MACHINE-LEARNING-SERVICE/src/Exceptions/server.exception.js

```diff
@@ -0,0 +1,28 @@
+/*
+@server.exception.js Copyright (c) 2022 Jalasoft
+CI 26 Sur #48-41, Ayurá Center, Edificio Unión № 1376, Medellín, Colombia
+All rights reserved
+This software is the confidential and proprietary information of
+Jalasoft Confidential Information You shall not
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
+class ServerException extends Error {
```

Comment:
Should be a line break between 12 and 13.
Add a comment for the class ServerException. What is it for?